moveit_planners/ompl/CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ install(TARGETS moveit_ompl_interface moveit_ompl_planner_plugin
   LIBRARY DESTINATION lib
   ARCHIVE DESTINATION lib
   RUNTIME DESTINATION bin
-  INCLUDES DESTINATION include
+  INCLUDES DESTINATION include/moveit_ros_ompl
```

Comment:
Similar to my other comments, I believe this should be `moveit_planners_ompl` to match the project name.